### PR TITLE
Newsletter January 2023

### PR DESCRIPTION
**Pull request by domidomi**
image

### CODE REVIEW

File: newsletter/v221.1.0-January-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# January 2023 StyleGuide newsletter
+
+Combined update notes, summarising changes in the style guide since the last newsletter (since the StyleGuide v219.1.0). Some of the key highlights include:
+
+- [StyleGuide Live Editor](<#-styleguide-live-editor-(beta-version)>)
```
Comment (katarzynatobis) [inline newsletter/v221.1.0-January-2023.mdx:5]: Wrong link, some weird characters :)

File: newsletter/v221.1.0-January-2023.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+## 🖼 Dialog for ads
+
+We have reworked Dialog layout so it holds slots for ads.
+
+![dialog for ads](./newsletter-assets/2023-january/dialog-ads.png)
+
+## 📈 StyleGuide Metrics
+
+We've created a tool that visualizes StyleGuide components usage.
```
Comment (katarzynatobis) [inline newsletter/v221.1.0-January-2023.mdx:62]: I would add that this is internal, only for Brainly (both access and data).

File: newsletter/v221.1.0-January-2023.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+Components using `focusStyle`: `Accordion`, `Avatar`, `Button`, `Dialog`, `FileHandler`, `HomeButton`, `IconAsButton`, `Label`, `Rating`, `Search`, `SkipLink`, native interactive elements besides `<a>`.
+
+<p float="left">
+  <img
+    src="./newsletter-assets/2023-january/focus-style-button.png"
+    width="200px"
+  />
+</p>
+
+Components using `focusTextStyle`: `Link`, `Checkbox`, `Radio`, `Select`, `Dropdown`, `Search`, `Textarea`, native `<a>`.
```
Comment (katarzynatobis) [inline newsletter/v221.1.0-January-2023.mdx:38]: Only Link, Dropdown, and <a> use `focusTextStyle`. My bad, I didn't update the description after changes 🙈

File: newsletter/v221.1.0-January-2023.mdx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+- added responsive props
+- animations no longer play on items expanded by default
+- increased performance of expanding and focusing Accordion items
+- added `aria-label` and `aria-labelledby` to Accordion
+
+### Button
+
+- `type` prop renamed to `variant` and `nativeType` renamed to `type`. This change allows the developer to use the `type` attribute of the native button element.
+- added high contrast mode for Button (#2540)
```
Comment (katarzynatobis) [inline newsletter/v221.1.0-January-2023.mdx:84]: PR number was overlooked, I think it should be removed

**Review by katarzynatobis — CHANGES_REQUESTED** [review]

**Review by coderitual — APPROVED** [review]

**Review by clxandstuff — APPROVED** [review]

**Review by katarzynatobis — APPROVED** [review]

**Pull request merged**